### ISSUE TITLE
Feature Parity: Pink background for exceptions/tracebacks?

### ISSUE THREAD

**Issue by mgeier** [jupyterlab/jupyterlab#7892]
## Description

JupyterLab has a pink background:

![image](https://user-images.githubusercontent.com/705404/74686039-87b42900-51d0-11ea-9b43-f54db9035594.png)

The Classic Notebook doesn't:

![image](https://user-images.githubusercontent.com/705404/74686102-bd591200-51d0-11ea-96a9-cfcaabf1b325.png)

## Reproduce

1. Run some code that raises an exception

## Expected behavior

Both should have the same background.

To me, it looks nicer without the background, because the pink color clashes with the text colors in the traceback, but this is of course a matter of taste.

## Context

If I'm not mistaken, the pink background was introduced in #1709 (more specifically in https://github.com/jupyterlab/jupyterlab/commit/a8b94adddabac196c630de9c5723c2e4ba0b1550) by @blink1073.

I don't know what the reason for this change was (or if there was a reason at all).

If the change was justified, it should be back-ported to the Classic Notebook, if not, it should be reverted.

**Comment by vidartf** [comment]
Note that there is no stated goal of having lab and classic notebook look exactly the same, so I don't think this should be backported to classic notebook even if we keep it here (there are other visual differences as well). That should not prevent us from having a discussion about how to best format error outputs though! The argument about low color contrast between text and background is the most interesting to consider here, especially since lab is looking to become more accessible.

**Comment by mgeier** [comment]
I've created a PR to fix this: #9420

Would be great if we could get this into the upcoming 3.0 release!

**Comment by krassowski** [comment]
Without the background it is:
- more readable and accessible (+)
- much less prominent when trying to scroll notebook searching for exception (-)
  - could we have something in between, with no background for text but a bold stripe indicating that there is an exception on a side? for example: `border-right: 5px solid pink` (could be left as well, or something very much different which would attract attention equally well).

**Comment by etep** [comment]
Hi Jupyter community, @JasonWeill , et al.,

The "pinkish red" background for `stderr` is sub-optimal.

There is a well known pattern where input and output are passed around via stdin and stdout which enables piped command lines like the following:
`cat some.pcap | some-binary | some-other-binary > output.pcap`
In this scheme (not my use case for Jupyter!), if any program has log information to print, it writes to stderr. I think this:
[Why python logging writes to stderr.](https://stackoverflow.com/questions/58971197/why-python-logging-writes-to-stderr-by-default) neatly encapsulates what I'm putting forward here.

In the case of a program error raising an exception, it makes sense to me to use the red background as a color scheme: In the error in fact case, the traceback information likely warrants extra "visual attention." If it is possible to treat `stderr` separately from literal program errors, this would be best imho. 

Succinctly, I see the background color for `stderr` as being a great candidate for local customization. In Python and in Jupyter, I use colored logs and those are written to stderr. The red background is not aesthetic and it is also confusing, i.e. conflated with actual program errors.

Maybe this is already a configurable param somewhere, but if not, please consider opening the issue and creating some config so that the default red background for stderr can be changed.

**Comment by JasonWeill** [comment]
I agree that using the pink/red color for "error" is not ideal. Besides the fact that `stderr` is not always used strictly for errors, as @etep mentions above, in East Asian cultures, red is often a good or lucky color. In Japan, for example, stock and related prices are shown in green or blue when they decline, or red when they increase, as in the following image from the Tokyo Stock Exchange:

![image](https://media-cldnry.s-nbcnews.com/image/upload/t_fit-1000w,f_auto,q_auto:best/rockcms/2024-08/20240805-tokyo-stocks-jj-1217a-1cf63c.jpg)

It would be good to make the `stderr` color customizable.

**Comment by jasongrout-db** [comment]
> In Japan, for example, stock and related prices are shown in green or blue when they decline, or red when they increase, as in the following image from the Tokyo Stock Exchange:

That's an awesome example of how different cultures use colors in different ways. Thanks for sharing!
